extensions/resteasy-reactive/rest/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/CompressionScanner.java

```diff
@@ -38,7 +38,8 @@ public List<HandlerChainCustomizer> scan(MethodInfo method, ClassInfo actualEndp
             return Collections.emptyList();
         }
 
-        AnnotationStore annotationStore = (AnnotationStore) methodContext.get(EndpointIndexer.METHOD_CONTEXT_ANNOTATION_STORE);
+        final AnnotationStore annotationStore = (AnnotationStore) methodContext
+                .get(EndpointIndexer.METHOD_CONTEXT_ANNOTATION_STORE);
```

Comment:
I suppose you will have to push a new version to address some of my concerns. If so, please let's not include this change. If not, no need to bother.